src/tcp.c

```diff
@@ -166,17 +183,16 @@ enftun_tcp_connect(struct enftun_tcp* tcp,
 }
 
 int
-enftun_tcp_connect_any(struct enftun_tcp* tcp,
-                       int mark,
-                       const char** hosts,
-                       const char* port)
+enftun_tcp_native_connect_any(struct enftun_tcp_native* ctx,
```

Comment:
Ditto here: `ctx` back to `tcp`.

Reply:
Is `connect_any` actually specific to the `_native` type?  If not, keep it as as `enftun_tcp_connect_any`.

Reply:
We could change `connect_any` to accept a `void*` for the first argument and then get the connect operation and pass that first argument in instead (since connect sometimes needs data besides what is in `enftun_tcp`).

Reply:
Will it not work with the first parameter as `struct enftun_tcp*`?  We always access `_connect` via `ops.connect`, which does have the first parameter as void.

Reply:
Aha yes, sorry, it casts it automatically and will find the variables in the correct spot since we have the `enftun_tcp` struct first.